src/Pages/Playlists/Components/VideoCart.js

```diff
@@ -0,0 +1,47 @@
+import { usePlaylist } from "../../../Context";
+import { removeVideoFromPlaylist } from "../../../ApiService";
+export const VideoCart = ({
+  _id,
+  title,
+  author,
+  thumnailMedium,
+  playlistId,
+}) => {
+  const { playlistDispatch } = usePlaylist();
+
+  const removeHandler = async (id, playlistId) => {
+    const encodeToken = localStorage.getItem("encodedToken");
+
+    const {
+      data: { playlist },
+      status,
+    } = await removeVideoFromPlaylist(playlistId, id, encodeToken);
+
+    if (status === 200 || status === 201) {
+      playlistDispatch({
+        type: "REMOVE_VIDEO_FROM_PLAYLIST",
+        videoId: id,
+        playlistId: playlistId,
+      });
+    }
+  };
+  const titleSet = title.length > 20 ? title.slice(0, 20) + "..." : title;
```

Comment:
For long titles instead of doing this, you can simply use the ellipsis property of text-overflow in CSS. it will handle this automatically. Read about it.